client/reader/discover/discover-stream.js

```diff
@@ -1,131 +1,36 @@
-import page from '@automattic/calypso-router';
-import { addLocaleToPathLocaleInFront, useLocale } from '@automattic/i18n-utils';
 import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
-import NavigationHeader from 'calypso/components/navigation-header';
-import { addQueryArgs } from 'calypso/lib/url';
 import withDimensions from 'calypso/lib/with-dimensions';
-import ReaderBackButton from 'calypso/reader/components/back-button';
 import ReaderMain from 'calypso/reader/components/reader-main';
 import DiscoverAddNew from 'calypso/reader/discover/components/add-new';
-import DiscoverNavigation from 'calypso/reader/discover/components/navigation';
+import DiscoverHeaderAndNavigation from 'calypso/reader/discover/components/header-and-navigation';
 import Reddit from 'calypso/reader/discover/components/reddit';
-import DiscoverTagsNavigation from 'calypso/reader/discover/components/tags-navigation';
-import Stream, { WIDE_DISPLAY_CUTOFF } from 'calypso/reader/stream';
+import Stream from 'calypso/reader/stream';
 import { useSelector } from 'calypso/state';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import { getReaderFollowedTags } from 'calypso/state/reader/tags/selectors';
 import {
 	getDiscoverStreamTags,
 	DEFAULT_TAB,
-	getSelectedTabTitle,
 	buildDiscoverStreamKey,
-	FIRST_POSTS_TAB,
 	ADD_NEW_TAB,
 	REDDIT_TAB,
 } from './helper';
-import './style.scss';
-
-const DISCOVER_HEADER_NAVIGATION_ITEMS = [];
-
-export const DiscoverHeader = ( props ) => {
```

Comment:
Moved to `DiscoverHeaderAndNavigation`. Here we have it as a separate component while in `DiscoverHeaderAndNavigation` I have combined it with the navigation.

Reply:
@mehmoodak I am getting linter errors in another PR https://github.com/Automattic/wp-calypso/pull/102702, related to `DiscoverHeader` not being available, looks like another file is expecting this component still, and needs a change too? - https://github.com/Automattic/wp-calypso/blob/c53896d8dad76815921285bb3aa8ee376a763d89/client/reader/discover/index.node.js#L9

Reply:
>  needs a change too?

Yes, indeed. Not sure why it didn't give the linting errors on this PR.

Reply:
PR is up to fix this. Thanks for the report 👍.